lib/metric.js

```diff
@@ -0,0 +1,26 @@
+function Metric() {
+  this.count = 0;
+  this.time = 0;
+  this.startTime = undefined;
+}
+
+Metric.prototype.start = function() {
+  this.startTime = process.hrtime();
+  this.count++;
+};
+
+Metric.prototype.stop = function() {
+  var now = process.hrtime();
```

Comment:
if you pass the `this.startTime` in, `process.hrtime` will only give you the delta and then you don't have to subtract below.

```
❯ node
> start = process.hrtime()
[ 182921, 98303035 ]
> delta = process.hrtime(start)
[ 3, 205637762 ]
> delta[0] * 1e9 + delta[1]
3205637762
>
```